### PR TITLE
Add supported browsers

### PR DESCRIPTION
**Pull request by AnaelMobilia**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | none

Add supported browsers.
TinyMce v4.7.1 [implies to have IE > 10](https://www.tinymce.com/docs/get-started/system-requirements/).
Using a generic formulas for others browsers frequently updated by their editors

The goal of this PR is to give guidelines to supported version (and visibility :-))

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ It is distributed under the GNU GENERAL PUBLIC LICENSE Version 2 - please consul
 * A web server (Apache, Nginx, IIS, etc.)
 * MariaDB (or MySQL < 5.7)
 * PHP 5.6 or higher
-* Mandatory PHP extensions:
+* Mandatory PHP extensions :
```
Comment (orthagh) [inline README.md]: no space before colon in EN

File: README.md [hunk 2]
```diff
@@ -46,6 +46,13 @@ It is distributed under the GNU GENERAL PUBLIC LICENSE Version 2 - please consul
     - ldap (users authentication)
     - openssl (encrypted communication)
 
+ * Supported browsers :
+    - IE 11+
+    - Edge
+    - Firefox (including ESR)
```
Comment (orthagh) [inline README.md]: For firefox, we could enforce that we support (the two) last ESR. At the moment, ESR45 is tested.
We have some css features who need recent version, i can't remember which version is needed but 45 is working.

File: README.md (hunk 1 above)
Comment (AnaelMobilia) [reply README.md]: Thanks !

File: README.md (hunk 2 above)
Comment (AnaelMobilia) [reply README.md]: Updated :)

**Pull request merged**